src/verifier.h

```diff
@@ -30,7 +30,7 @@ void VerifyWesolowskiProof(integer &D, form x, form y, form proof, uint64_t iter
 
 // Used only to verify 'Proof' objects in tests. This is not used by chia-blockchain.
 
-integer ConvertBytesToInt(uint8_t *bytes, int start_index, int end_index)
+integer ConvertBytesToInt(uint8_t const* bytes, int start_index, int end_index)
```

Comment:
I think it's more conventional and readable to put `const` _before_ the data type rather than after it. The same comment applies to other similar cases in this commit.

Reply:
I suppose this is a `east const` vs. `const west` issue. I happen to be in the former camp, that the type should come before the qualifier. I can change it though, if we use the `east const` convention

Reply:
I think all of these parameters really should be `uint8_t const* const bytes`, which (I think) highlights the value of consistently putting `const` on the right